pages/docs/concepts/asyncapi-document/reusing-traits.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Reusing Traits
```

Comment:
it is not about reusing traits

maybe better `Messages and Operations reusability with Traits`?

Reply:
For the name dont you think `Messages and Operations reusability with Traits` is gonna be too long? How about only `Traits`??